helm/generated_examples/baremetal-cleanbyjobs.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   name: local-static-provisioner-config
   namespace: default
   labels:
-    helm.sh/chart: provisioner-2.6.0-alpha.1
+    helm.sh/chart: provisioner-2.5.0-alpha.2
```

Comment:
why moved to lower version?

Reply:
we didn't release v2.5.0 which is the next minor version, I was thinking we should use v2.5.0 one instead of v2.6.0, the story around it is that tags were automatically created once the was a bump in these files.

I could release v2.6.0 and skip 2.5.0 too, then we'd no longer have the lint error in the presubmits

Reply:
after an offline conversation we changed it to v2.6.0-alpha.2, we're going to skip the release of v2.5.0 and release v2.6.0 directly